airflow/providers/databricks/hooks/databricks.py

```diff
@@ -22,25 +22,14 @@
 operators talk to the ``api/2.0/jobs/runs/submit``
```

Comment:
Does it precisely talks to **2.0** or **2.x**,
I am asking this because 2.1 is already out.